content/topics/analytics/_index.md

```diff
@@ -32,7 +32,7 @@ legislation:
 # Featured resource to at the top of the page
```

Comment:
### Fix

Update wording to be more clear

`Featured resource at the top of the page`